Dockerfile

```diff
@@ -28,4 +27,4 @@ LABEL io.k8s.display-name="o11y-exporters"
 LABEL io.openshift.tags="konflux"
 LABEL summary="Konflux Observability Exporters"
 
-CMD ["/bin/exporters"]
+ENTRYPOINT ["/bin/exporters"]
```

Comment:
We can probably revert this as we're not using args for now

Reply:
updated